pages/Frequency/Identity.md

```diff
@@ -0,0 +1,43 @@
+# Identity
+
+## Purpose
+
+1. Specify how DSNP Identifiers map to Frequency.
+1. Specify how the DSNP delegation maps to Frequency.
+1. Detail how Frequency matches the ownership requirements of DSNP.
+
+## Details
+
+DSNP uses [Message Source Accounts (MSAs)](https://libertydsnp.github.io/frequency/pallet_msa/index.html) that map directly to a DSNP Identity.
+
+## Identifier
+
+MSAs each receive a unique unsigned 64 bit identifier that is used as the DSNP Id.
+These MSAs are pseudo-anonymous identifiers that are associated with one or more public keys.
+MSAs can be created by the user or on behalf of the user using a signed authorization.
+
+### Creating a DSNP User Id
+
+1. Generate a [compatible key pair](https://wiki.polkadot.network/docs/learn-keys) in your wallet.
+1. Create a Message Source Account (MSA) on Frequency using one of the `create` extrinsics.
```

Comment:
should it link to https://libertydsnp.github.io/frequency/pallet_msa/index.html#dispatchable-functions ?